### PR TITLE
feat/yandex_metirca: onbaording yandex_metrica

### PR DESCRIPTION
**Pull request by ujjwal-ab**
## Description of the change

>onbaording yandex_metrica

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

- - -
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/687)

### CODE REVIEW

File: utils/client_server_name.js [hunk 1]
```diff
@@ -61,6 +61,7 @@ const clientToServerNames = {
   JUNE: 'June',
   ENGAGE: 'Engage',
   ITERABLE: 'Iterable',
+  YANDEX_METRICA: 'YandexMetrica',
```
Comment (MoumitaM) [inline utils/client_server_name.js:64]: Is `YandexMetrica` the display name?

Comment (ujjwal-ab) [reply utils/client_server_name.js:64]: No, It's Yandex.Metrica

Comment (MoumitaM) [reply utils/client_server_name.js:64]: client to server name mapping means config backend name to display name. Here you should put the display name.

Comment (ujjwal-ab) [reply utils/client_server_name.js:64]: Okay, Correcting it.

Comment (ujjwal-ab) [reply utils/client_server_name.js]: Done

**Review by MoumitaM — APPROVED** [review]

**Review by ItsSudip — APPROVED** [review]

**Pull request merged**